javascript/ql/src/semmle/javascript/Stmt.qll

```diff
@@ -145,7 +214,16 @@ private class MaybeDirective extends ExprStmt {
   }
 }
 
-/** A directive: string literal expression statement in the beginning of a statement container. */
+/**
+ * A directive: string literal expression statement in the beginning of a statement container.
+ *
+ * Examples:
+ *
+ * ```
+ * "use strict";
```

Comment:
Perhaps enclose this in a function body so it is clearer what a statement container is.